### DISCUSSION TITLE
Any equivalent to Ionic's modalController?

### DISCUSSION THREAD

**Discussion by beard7** [epicmaxco/vuestic-ui#4405] [Q&A]
Is there any way to achieve something similar to Ionic's modalController? It allows you to display a modal using a component for the content and pass properties.
You can then emit events from the modal, e.g. close or confirm, with additional data from the modal component.
The modalController is async and can await for the data emmited by the component when dismissed.

**Comment by m0ksem** [comment]
As for now, we don't even have `component` prop in `VaModal` and expect user to use slots. 

It is something that can be a feature request.
